js/bybit.js

```diff
@@ -952,26 +952,25 @@ module.exports = class bybit extends Exchange {
         const timestamp = undefined;
         const marketId = this.safeString (ticker, 'symbol');
         const symbol = this.safeSymbol (marketId, market);
-        const last = this.safeNumber (ticker, 'last_price');
-        const open = this.safeNumber (ticker, 'prev_price_24h');
-        let percentage = this.safeNumber (ticker, 'price_24h_pcnt');
+        const last = this.safeString (ticker, 'last_price');
+        const open = this.safeString (ticker, 'prev_price_24h');
+        let percentage = this.safeString (ticker, 'price_24h_pcnt');
         if (percentage !== undefined) {
-            percentage *= 100;
+            percentage = Precise.stringMul (percentage, '100');
         }
-        const baseVolume = this.safeNumber (ticker, 'turnover_24h');
-        const quoteVolume = this.safeNumber (ticker, 'volume_24h');
-        const vwap = this.vwap (baseVolume, quoteVolume);
+        const baseVolume = this.safeString (ticker, 'turnover_24h');
```

Comment:
Precise.stringMul does the check for `undefined` arguments, so we don't need an if-conditional here, the correct code is:
```JavaScript
        let percentage = this.safeString (ticker, 'price_24h_pcnt');
        if (percentage !== undefined) {
            percentage = Precise.stringMul (percentage, '100');
        }
```
↓
```JavaScript
        const percentage = Precise.stringMul (this.safeString (ticker, 'price_24h_pcnt'), '100');
```